certify-core/src/main/java/io/mosip/certify/core/entity/SvgTemplate.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
The POJO/DTOs package aren't present in certify-service package.

Reply:
I think it was done like that because we moved code from eSignet and the DTOs were created there because it has multiple sub project consuming from core lib.
But here in certify-service, i don't see the necessity for the same.
I also created a task on this.
https://mosip.atlassian.net/browse/INJICERT-679

Since the entity and respository are first time created, lets start in certify-service.

Reply:
All class to be moved out of services package.